modules/gitops-release-notes-1-11-2.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="release-notes-for-gitops-title-1-11-2_{context}"]
+= Release notes for {gitops-title} 1.11.2
+
+{gitops-title} 1.11.2 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-11-2_{context}"]
+== Errata updates
+
+[id="rhsa-2024-128925-02-gitops-1-11-2-security-update-advisory_{context}"]
```

Comment:
@Dhruv-Soni11 FYI and FYA:

NOTICE: [RHSA-2024:128925-02](https://errata.engineering.redhat.com/advisory/128925) has changed its advisory name to [RHSA-2024:1346-02](https://errata.engineering.redhat.com/advisory/128925)

Please update the PR and ping me, I will check and merge the RN and publish it. Thank you!

Reply:
Fixed it @Srivaralakshmi. PTAL.